jwql/utils/preview_image.py

```diff
@@ -330,8 +336,15 @@ def make_figure(self, image, integration_number, min_value, max_value,
                 format_string = "%.{}f".format(dig)
                 tlabelstr = [format_string % number for number in tlabelflt]
                 cbar = fig.colorbar(cax, ticks=tickvals)
+
+                # This seems to correctly remove the ticks and labels we want to remove. It gives a warning that
+                # it doesn't work on log scales, which we don't care about. So let's ignore that warning.
+                warnings.filterwarnings("ignore", message="AutoMinorLocator does not work with logarithmic scale")
```

Comment:
@bhilbert4 I am assuming we aren't too worried about the log scaling warning here because we are just trying to remove the minor ticks all together?

Reply:
This removes the default set of minor ticks and tick labels, so that all that remains is the custom set of ticks and tick labels.

Reply:
This is the only way I found to remove the defaults. Not ideal given the warning, but it works. My Issue on the matplotlib repo didn't really result in a useful answer.

Reply:
@bhilbert4 Yeah I remember seeing the back and forth resulting in nothing. This will have to do for now, we can always iterate in the future. Merging.